tests/test_ses/test_ses_boto3.py

```diff
@@ -243,6 +253,23 @@ def test_send_raw_email_validate_domain():
     sent_count.should.equal(2)
 
 
+@mock_ses
+def test_send_raw_email_invalid_address():
+    conn = boto3.client("ses", region_name="us-east-1")
+
+    message = get_raw_email()
+    del message["To"]
+    conn.verify_domain_identity(Domain="example.com")
+
+    invalid_addresses = ["test@example.com", "invalid_address"]
+    kwargs = dict(
+        Source=message["From"],
+        Destinations=invalid_addresses,
+        RawMessage={"Data": message.as_string()},
+    )
+    conn.send_raw_email.when.called_with(**kwargs).should.throw(ClientError)
```

Comment:
This needs the `pytest.raises` approach as well

Reply:
Fixed to `pytest.raises` format